docs/client-side/yeoman.rst

```diff
@@ -0,0 +1,260 @@
+Building Projects with Yeoman
+=============================
+By :ref:`Noel Rice <yeoman-author>` | Originally Published: 30 April 2015 
+
+`Yeoman <http://yeoman.io/>`_ builds a complete, running project for a given set of client-tools. You may be overwhelmed by "feature-shock" for the latest glut of client-tools (NPM, Gulp, Angular, etc.) or have simply run out of time to work through each feature set, much less the details, of every client tool. Yeoman generates everything you need to get over your first hurdle, the "hello world" that demonstrates that a group of technologies function together. 
+
+Yeoman is an open-source tool that works like a Visual Studio project template, but targets a wide developer audience that range from Ruby, browser extensions, PhoneGap, FaceBook React, jQuery-Mobile, and Microsoft technologies like ASP.NET.
+
+Yeoman is *opinionated*, that is, it prescribes tools and best practices for your target technologies so you don't have to decide "Use version X or Y?" or "What directory structure should I use?" Yeoman gets you stared with a known-good project that runs.
+
+The Yeoman command line tool yo works alongside a Yeoman generator. Generators define the technologies that go into a project. Here are a few sample `generators <http://yeoman.io/generators/>`_:
+
+- `AngularJS Generator <https://github.com/yeoman/generator-angular>`_ creates a starting point for a new single-page Angular application. 
+- `JQuery Generator <https://github.com/yeoman/generator-jquery>`_ creates the shell code for a jQuery plug-in.
+- `Chrome App Generator <https://github.com/yeoman/generator-chromeapp>`_ generates everything you need to create an extension for the Chrome browser.
+- The `Flux-React generator <https://github.com/banderson/generator-flux-react>`_ creates an application based on Facebook's Flux/React architecture.
+- `ASPNET generator <https://github.com/OmniSharp/generator-aspnet>`_ builds several types of ASP.NET 5 applications including Empty, Console, Web, Web API, Nancy ASP.NET, and Class Library.
+- You can even `generate your own Yeoman generator <https://github.com/yeoman/generator-generator>`_.
+
+In this article:
+	- `Getting Started`_
+	- `Integrating Generated Projects with Visual Studio`_
+	- `Generating a Webapp`_
+	- `Generating a Chrome Plugin`_
+
+Getting Started
+---------------
+Let's start by installing Yeoman without relying on the Visual Studio installation. 
+
+1.	Download and install `Node.js <https://nodejs.org/>`_ from https://nodejs.org. Leave the defaults to install the npm package manager and to make Node and npm accessible globally.
+
+.. image:: yeoman/_static/yeoman-nodejs-setup.png
+ 
+2.	Open a command line such as Windows PowerShell, and run the command to list all NPM packages found globally. The list will also show you where the packages are located on your system.
+
+.. code-block:: console
+
+ npm ls -g
+  
+3.	Run the command to install the Gulp command line interpreter:
```

Comment:
I want to avoid maintaining our own copy of the Yeoman getting started steps that is different than the ones on the Yeoman website (http://yeoman.io/learning/). Can we just say something like "Follow the instructions on http://yeoman.io/learning/ to install `yo` and other required tools" and then tell them to run the exact npm from the Yeoman site?: `npm install -g yo bower grunt-cli gulp`